lib/widgets/selection.dart

```diff
@@ -1212,8 +1220,10 @@ class _SelectionCounterState extends State<SelectionCounter> with SelectionHandl
 //*********** Queue actions ***********
 
 /// Action that queues a [Song] or a [SongOrigin] to be played next.
-class _PlayNextSelectionAction<T extends Content> extends StatelessWidget {
-  const _PlayNextSelectionAction({Key? key}) : super(key: key);
+class _PlayNextSelectionAction extends StatelessWidget {
+  final ContentType? contentType;
+
+  const _PlayNextSelectionAction({this.contentType, Key? key}) : super(key: key);
```

Comment:
I follow a style of that a constructors are at the top of the class

Reply:
There is a linter for this: [`sort_constructors_first`](https://dart-lang.github.io/linter/lints/sort_constructors_first.html). We should definitely enable it, but not in this PR, because a bunch of other classes don't follow the style already.

Reply:
The problem is that in some cases it actually is more valid and readable to have the constructors go last - e.g. see `Song` and other models.

Reply:
Yeah, I usually always put the member and fields first, I find it more readable. Let's not enable the lint rule then, you just going to have to tell me when I forget. 😃